build/extension_content_script.js

```diff
@@ -10,18 +10,25 @@ function log( str ) {
 	window.console.info( 'Who Wrote That? (Browser extension): ' + str );
 }
 /**
- * injectScript - Inject internal script to available access to the `window`
+ * Inject the extension's content script to the DOM, so that it's got access to the `window` object.
  *
  * @param  {string} filePath Local path of the internal script.
  * @param  {string} tag The tag as string, where the script will be append (default: 'body').
  * @see    {@link http://stackoverflow.com/questions/20499994/access-window-variable-from-content-script}
  */
 function injectScript( filePath, tag ) {
-	var node = document.getElementsByTagName( tag )[ 0 ],
-		script = document.createElement( 'script' );
-
-	script.setAttribute( 'type', 'text/javascript' );
-	script.setAttribute( 'src', filePath );
+	const wwtScriptId = 'wwt-content-script',
+		existingScript = document.getElementById( wwtScriptId );
+	if ( existingScript ) {
+		// Already injected.
+		return;
+	}
+	// Otherwise, inject a new script element.
+	let script = document.createElement( 'script' ),
+		node = document.getElementsByTagName( tag )[ 0 ];
+	script.id = wwtScriptId;
```

Comment:
`script` is an element; why not use `.setAttribute( ... )` like it did before? isn't the direct access to the properties less safe (in terms of behavior)?

Reply:
As I mentioned in Slack, I had a memory that best practice was to avoid `setAttribute()` in most cases, but I forgot why I remembered that. [This SO answer](https://stackoverflow.com/questions/10280250/getattribute-versus-element-object-properties) has some explanation and points to [more info](https://javascript.info/dom-attributes-and-properties).